src/main.rs

```diff
@@ -162,12 +162,9 @@ async fn listen_blocks(
     // Owerride S3 endpoint in case you want to use custom solution
     // like Minio or Localstack as a S3 compatible storage
     if let Some(s3_endpoint) = endpoint {
-        s3_conf = s3_conf.endpoint_resolver(Endpoint::immutable(s3_endpoint.parse::<Uri>().unwrap()));
-        tracing::info!(
-            target: INDEXER,
-            "Custom S3 endpoint used: {}",
-            s3_endpoint
-        );
+        s3_conf =
+            s3_conf.endpoint_resolver(Endpoint::immutable(s3_endpoint.parse::<Uri>().unwrap()));
```

Comment:
Was it considered to use `Uri` as `endpoint` field type in the RunArgs structure to avoid a need to parse and `.unwrap()` here?

Reply:
No, missed it. Will try.